package.json

```diff
@@ -35,6 +35,7 @@
     "client"
   ],
   "devDependencies": {
+    "@types/node-fetch": "^2.6.11",
```

Comment:
I don't think we need this types library for node-fetch; the types that we are using for node-fetch are coming directly from the node-fetch package